src/library/ListItem/Labels/Oversubscribed.tsx

```diff
@@ -46,7 +46,10 @@ export const Oversubscribed = ({
   const lowestReward = eraStakers?.lowestReward ?? 0;
 
   const displayOversubscribed =
-    synced.stake && totalNominations >= consts.maxNominatorRewardedPerValidator;
+    synced.stake &&
+    consts.maxNominatorRewardedPerValidator.isLessThanOrEqualTo(
```

Comment:
Same issue, logic is different. Should be `isLessThan`, and since the numbers are now both big numbers it does not need to change.